Dockerfile

```diff
@@ -12,24 +12,16 @@ ENV KBOT 1.9.1
 
 
 ### 2. Get Java and all required system libraries
-RUN apt-get update && apt-get upgrade -y \
- && apt-get install -y software-properties-common \
-  build-essential git \
-  openjdk-8-jre openjdk-8-jdk
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y software-properties-common build-essential openjdk-8-jre openjdk-8-jdk git make curl wget tar
 
-RUN apt-get update \
-    && apt-get install -y make curl git wget tar
-
-RUN pwd
 
 ### 3. Install custom tools
 WORKDIR /tools
 
-RUN pwd
-
 # Avoid repeated downloads of script dependencies by mounting the local coursier cache:
-# docker run -v $HOME/.coursier/cache/v1:/tools/.coursier-cache ...
-#ENV COURSIER_CACHE "/tools/.coursier-cache"
+#docker run -v $HOME/.coursier/cache/v1:/tools/.coursier-cache ...
+ENV COURSIER_CACHE "/tools/.coursier-cache"
```

Comment:
What is this used for?

Reply:
Tries to use the cache wherever possible and avoid redundant downloads. Inspiration: https://github.com/INCATools/ontology-development-kit/blob/739b82c38a2a45217977eba3d285e8fa94af13c9/Dockerfile#L67

Reply:
So you have this cache on your machine, or whoever builds the Docker image should have it  or has to have it?

Reply:
I am actually not sure how this works in context of the docker image. Looking at the docs, this is supposed to cache static resources (https://get-coursier.io/docs/next/cache) at the location specified, but since the location I am specifying is in the `WOKRDIR` which doesn't seem to get shared between containers, then I am not sure how it is working. I do see `using cache` in the logs.

Reading this (https://twg.io/blog/things-i-wish-i-knew-about-docker-before-i-started-using-it/) it looks like docker anyway caches, so `using cache` maybe because of that and not `coursier cache`, although when I commented it out, it stopped using the cache....

I need to figure this out..
<img width="821" alt="Screen Shot 2020-04-13 at 7 45 08 PM" src="https://user-images.githubusercontent.com/8756596/79171991-739c3a80-7dc1-11ea-89c9-cc91ecfe311b.png">

Reply:
Actually I misspoke slightly. It would have no role during building. But presumably, if you volume-mounted the cache to this place when you run the container it would get used, and I suppose the point is to cache from one run of the container to the next. The volume-mount is in this `docker run` example you linked to:
```
docker run -v $HOME/.coursier/cache/v1:/tools/.coursier-cache ...
```

Reply:
Oh okay.

Reply:
This cache is mostly there for Scala scripts run with the Ammonite script runner, which is included in the ODK. We don't really need it here at the moment.